<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WizardPage</class>
 <widget class="QWizardPage" name="WizardPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>886</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>531</width>
     <height>641</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>121</x>
       <y>168</y>
       <width>56</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>비밀번호</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="loginLineEdit_2">
     <property name="geometry">
      <rect>
       <x>183</x>
       <y>171</y>
       <width>142</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="SignUpPushButton">
     <property name="geometry">
      <rect>
       <x>331</x>
       <y>168</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>회원가입</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>100</y>
       <width>231</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>학생 로그인</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loginPushButton">
     <property name="geometry">
      <rect>
       <x>331</x>
       <y>131</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>로그인</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>121</x>
       <y>131</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>아이디</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="loginLineEdit">
     <property name="geometry">
      <rect>
       <x>183</x>
       <y>134</y>
       <width>142</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="idFindButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>260</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>id찾기</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pwFindButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>260</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>pw찾기</string>
     </property>
    </widget>
    <widget class="QLabel" name="loginLabel">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>220</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>환영함</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>90</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>학생용 클라이언트 회원가입</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>417</width>
       <height>206</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEdit_email">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_new_pw"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>비밀번호</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="EmailCheckPushButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>인증 요청</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_new_pw_check"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_new_id">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>아이디</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>비밀번호 확인</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>이메일</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>인증번호</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEdit_email_check">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="EmailCheckNumberPushButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>인증</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>이름</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_new_name"/>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="SignUpCheckButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>중복확인</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="BackButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>380</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>뒤로가기</string>
     </property>
    </widget>
    <widget class="QLabel" name="SignUpLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>420</y>
       <width>291</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>아이디와비밀번호를입력해주세요.</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SignUpPushButton_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>380</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>회원가입</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_7">
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>140</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>id찾기</string>
     </property>
    </widget>
    <widget class="QPushButton" name="idFindPageEmailButton">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>170</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>이메일 확인</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="idFindPageEmailLineEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>170</y>
       <width>201</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>이메일 입력</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_6">
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>190</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>아이디 입력</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>170</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>비밀번호 찾기</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pwFindPageEmailLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>220</y>
       <width>201</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="pwFindPageIdLineEdit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>190</y>
       <width>201</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>220</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>이메일 입력</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pwFindPageEmailButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>220</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>비밀번호 전송</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pwFindPageIdButton">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>190</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>아이디 확인</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>151</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>학생 메인화면</string>
     </property>
    </widget>
    <widget class="QPushButton" name="mainButton_1">
     <property name="geometry">
      <rect>
       <x>61</x>
       <y>221</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>등급</string>
     </property>
    </widget>
    <widget class="QPushButton" name="mainButton_2">
     <property name="geometry">
      <rect>
       <x>147</x>
       <y>221</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>문제</string>
     </property>
    </widget>
    <widget class="QPushButton" name="mainButton_3">
     <property name="geometry">
      <rect>
       <x>234</x>
       <y>222</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>상담</string>
     </property>
    </widget>
    <widget class="QPushButton" name="mainButton_4">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>225</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Q&amp;&amp;A</string>
     </property>
    </widget>
    <widget class="QPushButton" name="beackButton_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>494</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>뒤로가기</string>
     </property>
    </widget>
    <widget class="QLabel" name="userNameLabel">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>110</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>유저명</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QTextBrowser" name="logTextBrowser_2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>40</y>
     <width>331</width>
     <height>641</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 87, 83);
color: rgb(238, 238, 236);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>클라이언트Log</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
